sdks/java/io/mongodb/src/test/java/org/apache/beam/sdk/io/mongodb/MongoDBGridFSIOTest.java

```diff
@@ -82,14 +83,38 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(MongoDBGridFSIOTest.class);
 
   private static final String MONGODB_LOCATION = "target/mongodb";
-  private static final int PORT = 27017;
   private static final String DATABASE = "gridfs";
 
   private static transient MongodExecutable mongodExecutable;
 
+  private static int port;
+
+  /**
+   * Looking for an available network port.
+   *
+   * @param min The min port number.
+   * @param max The max port number.
+   * @return The available port number.
+   */
+  private static int availablePort(int min, int max) {
```

Comment:
Yup -- if we don't care for the port number, the system can give us one; we don't need to search for it.